### PR TITLE
Wc 88 calendar style changes

### PR DESCRIPTION
**Pull request by unjust**
#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-88

#### Description
style changes for our calendar component. see ticket for specs.

#### Screenshots (if applicable)
screen shot 2017-09-12 at 3 52 11 pm

### CODE REVIEW

File: assets/scss/components/_calendar.scss [hunk 1]
```diff
@@ -1 +1,55 @@
...
+  }
+
+.flatpickr-current-month {
+	width:80%;
+	& span.cur-month {
+		font-weight: $W_normal;
+	}
+}
+
+.flatpickr-day.selected,
```
Comment (mperrotti) [inline assets/scss/components/_calendar.scss]: This CSS is a little crazy. Is there a shorter/better way to select all of these things?

File: src/forms/CalendarComponent.jsx [hunk 2]
```diff
@@ -32,8 +32,18 @@ class CalendarComponent extends React.Component {
 			onClose: this.onClose,
 			altInput: true,
 			allowInput: true,
-			altFormat: 'D M d, Y', // TODO localize
+			altFormat: 'D M d, Y',
 			defaultDate: this.props.value,
+			nextArrow: `<span class="svg svg--chevron-right">
```
Comment (mperrotti) [inline src/forms/CalendarComponent.jsx]: Is there a reason we can't just render the `<Icon>` component instead of manually adding the markup?

Comment (unjust) [reply src/forms/CalendarComponent.jsx]: these config options for `nextArrow` and `prevArrow` expect `html`
https://chmln.github.io/flatpickr/options/

so I don't think the component will work. I'm pretty sure I tried it a while back.

File: assets/scss/components/_calendar.scss (hunk 1 above)
Comment (unjust) [reply assets/scss/components/_calendar.scss]: I know. But I took this selector right from their stylesheet. I imagine its like this for a reason.

**Comment by unjust** [comment]
@mperrotti didn't follow up and approve but we discussed and I think he verbally approved.

**Pull request merged**